docs/command-reference/strings/append.md

```diff
@@ -10,34 +10,33 @@ import PageTitle from '@site/src/components/PageTitle';
 
 ## Introduction
 
-The `APPEND` command in Redis/Dragonfly is used to add a specified value to the end of a string stored at a given key. If the key doesn't exist, it creates it and initializes it with an empty string before appending. This command is particularly useful for accumulating logs or constructing strings incrementally.
+In Dragonfly, as well as in Redis and Valkey, the `APPEND` command is used to add a specified value to the end of a string stored at a given key.
+If the key doesn't exist, the command creates a new key and initializes it with an empty string before appending.
+This command is particularly useful for accumulating logs or constructing strings incrementally.
 
 ## Syntax
 
-```cli
+```shell
 APPEND key value
 ```
 
 ## Parameter Explanations
 
-- `key`: The key of the string you wish to append to. It should already exist as a string type, or it will be created as a new string.
-- `value`: The string value to append to the existing string at the key.
+- `key`: The key of the string you want to append data to.
+  If the key already exists, it must be of type string; otherwise, a new string will be created.
+- `value`: The string value to be appended to the existing string associated with the key.
 
 ## Return Values
 
-- An integer indicating the length of the string after the append operation.
-
-**Example Outputs**
```

Comment:
I don't like the formatting for `Example Outputs`, it should be header-3 instead of bold. Also, I feel that these sections don't add a lot of value without context. As long as return values are properly explained, the upcoming code examples are more relevant.